platform/src/main/java/org/stellar/anchor/platform/data/JdbcSepTransaction.java

```diff
@@ -52,6 +54,11 @@ public abstract class JdbcSepTransaction {
   @Column(name = "amount_fee_asset")
   String amountFeeAsset;
 
+  @SerializedName("fee_details")
```

Comment:
Can we remove the `amountFee` and implement `setAmountFee` and `getAmountFee`  to get and set the `fee_details`?